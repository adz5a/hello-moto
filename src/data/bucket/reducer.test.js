import {
    reducer as bucket
} from "data/bucket";

test('reducers', () => {
    let state;
    state = bucket({bucket:{saved:false,baseURL:null,bucket:null,prefixes:{},content:[]}}, {type:'BUCKET/LIST-DIRS',data:{baseURL:'https://s3.eu-west-2.amazonaws.com',bucket:'imgs-yolo',prefixes:{content:{'content/':{}}}},meta:{origin:'middleware'}});

    expect(state).toMatchSnapshot();

});
