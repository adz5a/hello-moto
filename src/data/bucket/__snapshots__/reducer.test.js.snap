// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers 1`] = `
Object {
  "baseURL": "https://s3.eu-west-2.amazonaws.com",
  "bucket": "imgs-yolo",
  "prefixes": Object {
    "content": Object {
      "content/": Object {},
    },
  },
}
`;
